modules/tasks/tasks/src/main/java/org/motechproject/tasks/service/impl/TaskActivityServiceImpl.java

```diff
@@ -203,6 +216,21 @@ public long getAllTaskActivitiesCount(Set<TaskActivityType> activityTypes, Range
         return taskActivitiesDataService.countByActivityTypesAndDate(activityTypes, dateRange);
     }
 
+    @Transactional
```

Comment:
remove transactional, it does work on private method. It will be executed in transaction because it is invoked in public method annotated with transactional